nifi-registry-core/nifi-registry-framework/src/main/java/org/apache/nifi/registry/security/authorization/file/FileAccessPolicyProvider.java

```diff
@@ -147,6 +149,7 @@ private static JAXBContext initializeJaxbContext(final String contextPath) {
 
     static final String PROP_NIFI_IDENTITY_PREFIX = "NiFi Identity ";
     static final String PROP_USER_GROUP_PROVIDER = "User Group Provider";
+    static final String PROP_NIFI_IDENTITY_GROUP_NAME = "Identity Group Name";
```

Comment:
I think we should probably call this `NiFi Identity Group Name` since that would correspond with the other properties for `NiFi Identity`

Reply:
Initially i was going to call it `NiFi Identity Group` or `NiFi Identity Group Name` but then i realised that:
if no node identites are present, only NiFi Identity Group Name, then the `PROP_NIFI_IDENTITY_PREFIX` would regex match it, and would create user authorizations for it. Thus i opted to name it differently to avoid false instance `NiFi Identity` matches

Reply:
Ok that makes sense, how about just `NiFi Group Name` then?